src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx

```diff
@@ -28,10 +34,7 @@ minSdkVersion 24
 
 ## Web
 
-There are no Amplify specific requirements or setup instructions when targeting web. You will need to use a browser supported by Flutter. See the following Flutter docs for more info:
```

Comment:
Can we leave this? Sorry - I may have suggested the changes on the right. I didn't realize this was a file I had already updated.

Reply:
Would prefer to leave as well